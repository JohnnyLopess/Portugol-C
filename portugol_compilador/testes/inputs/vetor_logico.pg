programa {
    funcao inicio() {
        logico v[2]
        v[0] = 1
        v[1] = 0
        escreva(v[0])
        escreva(v[1])
    }
}
