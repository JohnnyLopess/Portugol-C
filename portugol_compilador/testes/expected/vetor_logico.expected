#include <stdio.h>

int main() {
    bool v[2];
    v[0] = 1;
    v[1] = 0;
    printf("%d\n", v[0]);
    printf("%d\n", v[1]);
    return 0;
}
